src/main/fc/rc_controls.c

```diff
@@ -118,11 +119,19 @@ throttleStatus_e calculateThrottleStatus(void)
     return THROTTLE_HIGH;
 }
 
+#define ARM_DELAY_MS        500
+#define STICK_DELAY_MS      50
+#define STICK_AUTOREPEAT_MS 250
+#define repeatAfter(t)      {rcDelayMs -= (t);doNotRepeat = false;}
 void processRcStickPositions(throttleStatus_e throttleStatus)
 {
-    static uint8_t rcDelayCommand;      // this indicates the number of time (multiple of RC measurement at 50Hz) the sticks must be maintained to run or switch off motors
-    static uint8_t rcSticks;            // this hold sticks position for command combos
-    static uint8_t rcDisarmTicks;       // this is an extra guard for disarming through switch to prevent that one frame can disarm it
+    // time the sticks are maintained
+    static int16_t rcDelayMs = 0;
```

Comment:
Please move these to global scope to avoid `static` initialization overhead :-)

Reply:
You are correct, static data are already valid only after first stick change so do not need initialization :oops: